### ISSUE TITLE
ARIA landmark roles should have an aria-labelledby attribute

### ISSUE THREAD

**Issue by jcivjan** [instructure/canvas-lms#50]
It would greatly help with accessibility if the landmark aria roles that are designating the different sections of your page had aria-labelledby attributes pointing to the appropriate headings.
